Sources/SpeziLLMOpenAI/Configuration/LLMOpenAIModelParameters.swift

```diff
@@ -51,7 +50,7 @@ public struct LLMOpenAIModelParameters: Sendable {
     ///   - logitBias: Alters specific token's likelihood in completion.
     ///   - user: Unique identifier for the end-user, aiding in abuse monitoring.
     public init(
-        responseFormat: ChatQuery.ResponseFormat? = nil,
+        responseFormat: Components.Schemas.CreateChatCompletionRequest.response_formatPayload? = nil,
```

Comment:
I am wondering if we should add compact type aliases for this?

Reply:
I’ve added an `LLMOpenAIRequestType` alias. Does that work for you?

Should we also introduce an alias for `Components.Schemas` in general? This won’t make the types shorter, but something like LLMOpenAIGeneratedTypes could improve readability, maybe?

Reply:
I think we can introduce well defined and named typealias for the specific types that we use in our API surface; we should see if we can make them compact and focus on them.